torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -976,6 +978,33 @@ def test_average_parameters(self):
                 for p in model.parameters():
                     self.assertEqual(p.data, torch.ones_like(p.data) * 0.5)
 
+        @unittest.skipIf(
+            BACKEND != "nccl" and BACKEND != "gloo",
+            "MPI backend does not support creating subgroups on CUDA devices",
+        )
+        @skip_if_lt_x_gpu(2)
+        @skip_if_odd_num_of_gpus()
+        def test_periodic_model_averager(self):
+            rank = dist.get_rank()
+            rank_to_GPU = self._init_multigpu_helper()
+            device_id = rank_to_GPU[rank][0]
+            world_size = dist.get_world_size()
+
+            model = nn.Linear(1, 5, bias=False).cuda(device_id)
+            param = next(model.parameters())
```

Comment:
any reason we check only one param vs the entire `model.parameters`?

Reply:
This is because there is only a single linear layer, so there is only a single parameter tensor.